BUILD.bazel

```diff
@@ -66,3 +75,17 @@ js_library(
     },
     visibility = ["//visibility:public"],
 )
+
+npm_package(
+    name = "npm_package",
+    srcs = [
+        "LICENSE",
+        "README.md",
+        "package.json",
+        "//tools:check_version.js",
+    ],
+    deps = [
+        "//internal:tsc_wrapped",
+        "//internal/karma:karma_concat_js",
```

Comment:
karma should be in its own npm package.
see my changes in https://github.com/alexeagle/rules_typescript/tree/distribute_js

Reply:
done